nilearn/_utils/niimg_conversions.py

```diff
@@ -120,7 +120,7 @@ def check_niimg(niimg, ensure_3d=False):
     ----------
     niimg: Niimg-like object
         See http://nilearn.github.io/building_blocks/manipulating_mr_images.html#niimg.
-        If niimg is a string, consider it as a path to Nifti image and
+        If niimg is a string, consider it as a path to nifti image and
```

Comment:
*a* nifti image or maybe Nifti image not sure what the convention about the capitalization of nifti is.